startup_scripts/includes/__init__.py

```diff
@@ -0,0 +1,11 @@
+from ruamel.yaml import YAML
+from pathlib import Path
+
+
+def load_yaml(yaml_file: str):
+    yf = Path(yaml_file)
```

Comment:
I believe that we should stick to one indention. You suggested to change the indention to four spaces in private. Though until we've done that everywhere we should stick to two spaces.